src/modules/Dashboard/components/TakeNowModal/TakeNowModal.tsx

```diff
@@ -101,11 +102,12 @@ export const useTakeNowModal = ({ dataTestId }: UseTakeNowModalProps) => {
       newActivityOrFlow?: Activity | HydratedActivityFlow | ParticipantActivityOrFlow,
     ) => {
       const props: MixpanelPayload = {
-        [MixpanelProps.Feature]: 'Multi-informant',
         [MixpanelProps.AppletId]: appletId,
         [MixpanelProps.MultiInformantAssessmentId]: multiInformantAssessmentId,
         ...payload,
       };
+      addFeatureToAnalyticsPayload(props, 'Multi-informant');
```

Comment:
This function feels like just a special case of spread appending, but I see that the inline syntax for this will be a bit awkward because the values in the payload are `unknown`. I'm doing a bit of a typed refactor that may make this function unnecessary

Reply:
I'm totally down for improvements here… I wasn't necessarily too pleased with what I came up with in the web app, either, so I'm all ears!

Reply:
Will have a PR up soon 🙏